### PR TITLE
Use autolinks from ReSpec where possible

### PR DESCRIPTION
**Pull request by dontcallmedom**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -211,7 +205,7 @@ <h2>
               </p>
               <p>
                 Captured audio from an element with an <a href=
-                "http://www.w3.org/TR/html5/embedded-content-0.html#effective-playback-rate">effective
+                "https://html.spec.whatwg.org/multipage/media.html#ended-playback#dom-media-playbackrate">effective
```
Comment (martinthomson) [inline index.html]: Two fragments?
```suggestion
                "https://html.spec.whatwg.org/multipage/media.html#dom-media-playbackrate">effective
```

**Review by martinthomson — COMMENTED** [review]
One small error in the manual conversion there.

Does this really need index-new.html?  Or is that an error too?

File: index.html (hunk 1 above)
Comment (dontcallmedom) [reply index.html]: fixed

**Comment by dontcallmedom** [comment]
not sure how index-new.html ended up there, but now fixed

**Pull request merged**